src/main/java/seedu/address/logic/Messages.java

```diff
@@ -24,7 +24,7 @@ public class Messages {
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
     public static final String MESSAGE_DUPLICATE_FIELDS =
                 "Multiple values specified for the following single-valued field(s): ";
-    public static final String MESSAGE_INVALID_ROLE = "Invalid role!";
+    public static final String MESSAGE_INVALID_ROLE = "Next-of-Kin are not available for Doctors and Nurses!";
```

Comment:
If I recall, some other commands also use this message. Will the change tally with them?

Reply:
You are right! Thanks for pointing it out will change it in the next commit.